src/areaCodeList.js

```diff
@@ -504,4 +504,4 @@ export const AREA_CODE_LIST = [
   '985',
   '986',
   '989',
-];
+]);
```

Comment:
Is this exported for consumers anywhere?

Reply:
No - it's (currently) intended only for internal use.  It is only referenced in the base.js and within the geo.test.js: https://github.com/search?q=repo%3Ahearsaycorp%2Fphonenumber-util%20AREA_CODE_LIST&type=code